resources/dicts/events/lifegen_events/ceremonies.json

```diff
@@ -203,6 +203,12 @@
         "The silver glow of the moon casts a gentle light over the Clan as they come together in silent anticipation. With pride evident in their gaze, c_l beckons you forward. Each step you take resonates with purpose and commitment. c_l's tail sweeps around you in a protective embrace, sealing your pledge to always safeguard and nurture the Clan's future. The air is thick with emotions as you are hailed the new permanent queen of c_nClan -- y_c.",
         "In the gentle embrace of dawn, the Clan surrounds you, their eyes reflecting hope and expectation. c_l steps to your side, their presence reassuring. With a shared breath, a bond forged from mutual respect and understanding, you step into your new role. Recognized for your compassion, wisdom, and maternal instincts, the Clan bursts into cheers, celebrating the rise of their new permanent queen y_c."
     ],
+    "queen_ceremony forgiven": [
+        "Despite your rocky history with the Clan, c_l feels confident in granting you your full name. Beneath the Great Rock, they name you y_c, and honor you for your honor1. MENTOR"
```

Comment:
is this supposed to have MENTOR and NOMENTOR still

Reply:
LMAO no it is not . one sec